lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -263,3 +246,59 @@ function dosomething_rogue_rb_exists_in_rogue($rbid)
 {
   return db_query("SELECT * FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE rbid = :rbid", array(':rbid' => $rbid))->fetchAll();
 }
+
+/**
+ * Transform reportback into the appropriate scheme to send to Rogue.
+ *
+ * @param $northstar_id - User's Northstar user id.
```

Comment:
since there is a small percentage of people that do not have NS Ids, should I do a check before this helper function to send `$northstar_id` if there is one and if not, send the `$user`?